tests/ha/single_pod_failure/test_data_pod_failure.py

```diff
@@ -155,7 +155,8 @@ def teardown_method(self):
                 resp = self.ha_obj.delete_s3_acc_buckets_objects(self.s3_clean)
             else:
                 resp = self.ha_obj.delete_s3_acc_buckets_objects(self.s3_clean, obj_crud=True)
-            assert_utils.assert_true(resp[0], resp[1])
+            if not resp[0]:
+                LOGGER.error("Failed to delete objects/buckets")
```

Comment:
This needs to be removed once we do not have to re-deploy after every test, right ?

So can we add TODO, accordingly ?

Reply:
No the thought was that in any case, we should go for at least restoring the pod or else rest of the tests of the suite will start failing. And as such deletes will be tested in deletes test cases. What u think?